editor/plugins/abstract_polygon_2d_editor.cpp

```diff
@@ -238,15 +305,18 @@ void AbstractPolygon2DEditor::disable_polygon_editing(bool p_disable, const Stri
 	button_create->set_disabled(p_disable);
 	button_edit->set_disabled(p_disable);
 	button_delete->set_disabled(p_disable);
+	button_center->set_disabled(p_disable);
 
 	if (p_disable) {
 		button_create->set_tooltip_text(p_reason);
 		button_edit->set_tooltip_text(p_reason);
 		button_delete->set_tooltip_text(p_reason);
+		button_center->set_tooltip_text(p_reason);
 	} else {
 		button_create->set_tooltip_text(TTR("Create points."));
 		button_edit->set_tooltip_text(TTR("Edit points.\nLMB: Move Point\nRMB: Erase Point"));
 		button_delete->set_tooltip_text(TTR("Erase points."));
+		button_center->set_tooltip_text(TTR("Move center of gravity to origin."));
```

Comment:
```suggestion
		button_center->set_tooltip_text(TTRC("Move center of gravity to origin."));
```
All of the buttons should use TTRC, but you don't need to update them.